packages/amplication-data-service-generator/Dockerfile

```diff
@@ -1,24 +1,32 @@
-# Use node 14.15.3 as the base image
-FROM node@sha256:bac289a6f393990e759c672d5f567553c697255d1fb858e2c62d086a2dfae44a AS node
-FROM node
+ARG ALPINE_VERSION=alpine3.14
```

Comment:
Make sure to rebase from master, this is outdated

Reply:
Done